packages/@aws-cdk/aws-elasticloadbalancingv2/lib/alb/application-target-group.ts

```diff
@@ -28,6 +28,13 @@ export interface ApplicationTargetGroupProps extends BaseTargetGroupProps {
    */
   readonly protocol?: ApplicationProtocol;
 
+  /**
+   * The protocol version to use
+   *
+   * @default - No protocol version
```

Comment:
This is one of those cases where I think while technically true that no protocol will be explicitly set, the more user-friendly documentation is to note that HTTP1 is the default.

```suggestion
   * @default ApplicationProtocolVersion.HTTP1
```

Reply:
That makes sense, thanks for clarifying!